iceoryx_posh/test/integrationtests/service_discovery/test_roudi_findservice.cpp

```diff
@@ -40,6 +40,70 @@ TEST_F(RoudiFindService_test, OfferSingleMethodServiceSingleInstance)
     ASSERT_THAT(*instanceContainer.begin(), Eq(IdString("instance1")));
 }
 
+TEST_F(RoudiFindService_test, OfferServiceWithDefaultDescription)
+{
+    senderRuntime->offerService(iox::capro::ServiceDescription());
+    this->InterOpWait();
+    iox::runtime::InstanceContainer instanceContainer;
+    receiverRuntime->findService(iox::capro::ServiceDescription(), instanceContainer);
+
+    ASSERT_THAT(instanceContainer.size(), Eq(1u));
+    senderRuntime->stopOfferService(iox::capro::ServiceDescription());
+}
+
+TEST_F(RoudiFindService_test, ReOfferSingleMethodServiceSingleInstance)
```

Comment:
Please adjust the test name to something more understandable. Are you expecting that a reoffered service can be found? Maybe a name like: ReofferedServiceWithValidServiceDescriptionCanBeFound

Reply:
i want to check for a sequence here, i.e. offer -> stopOffer -> offere, then may be naming  OfferStopofferOfferServiceSequence might hold good?

Reply:
@nihalchari I would prefer reoffer, I think the name already implies a sequence like offer -> stopOffer -> offer. And what is the expected outcome? Or how do you verify that the sequence works?